x-pack/plugins/security_solution/public/entity_analytics/pages/entity_analytics_management_page.tsx

```diff
@@ -5,38 +5,171 @@
  * 2.0.
  */
 
-import React from 'react';
-import { EuiBetaBadge, EuiFlexGroup, EuiFlexItem, EuiPageHeader, EuiSpacer } from '@elastic/eui';
-
+import React, { useState, useMemo } from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPageHeader,
+  EuiHorizontalRule,
+  EuiButton,
+  EuiText,
+  EuiSpacer,
+} from '@elastic/eui';
+import styled from '@emotion/styled';
+import { euiThemeVars } from '@kbn/ui-theme';
+import moment from 'moment';
 import { RiskScorePreviewSection } from '../components/risk_score_preview_section';
 import { RiskScoreEnableSection } from '../components/risk_score_enable_section';
 import { ENTITY_ANALYTICS_RISK_SCORE } from '../../app/translations';
-import { BETA } from '../../common/translations';
 import { RiskEnginePrivilegesCallOut } from '../components/risk_engine_privileges_callout';
 import { useMissingRiskEnginePrivileges } from '../hooks/use_missing_risk_engine_privileges';
+import { RiskScoreUsefulLinksSection } from '../components/risk_score_useful_links_section';
+import { RiskScoreConfigurationSection } from '../components/risk_score_configuration_section';
+import { useRiskEngineStatus } from '../api/hooks/use_risk_engine_status';
+import { useScheduleNowRiskEngineMutation } from '../api/hooks/use_schedule_now_risk_engine_mutation';
+import { useAppToasts } from '../../common/hooks/use_app_toasts';
+import * as i18n from '../translations';
 
 export const EntityAnalyticsManagementPage = () => {
   const privileges = useMissingRiskEnginePrivileges();
+  const [includeClosedAlerts, setIncludeClosedAlerts] = useState(false);
+  const [from, setFrom] = useState(localStorage.getItem('dateStart') || 'now-30m');
+  const [to, setTo] = useState(localStorage.getItem('dateEnd') || 'now');
+  const { data: riskEngineStatus } = useRiskEngineStatus();
+  const currentRiskEngineStatus = riskEngineStatus?.risk_engine_status;
+  const runEngineEnabled = currentRiskEngineStatus === 'ENABLED';
+  const [isLoading, setIsLoading] = useState(false);
+  const { mutate: scheduleNowRiskEngine } = useScheduleNowRiskEngineMutation();
+  const { addSuccess } = useAppToasts();
+  const VerticalSeparator = styled.div`
```

Comment:
If you decide to extract and re-use, could you also include the updates from my comment here? I’m happy to handle the other EUI updates, but if you’re already in the code, it would be great to tackle these while you’re here. 🙇

Reply:
Sure, @machadoum, I’ve set this up as a shared component, and it’s now being reused.  

Also, @CAWilson94, I’ll incorporate the updates from your comment. Thanks for pointing that out!